### ISSUE TITLE
invalid bytes32 value in empty state delta

### ISSUE THREAD

**Issue by lacabra** [enigmampc/enigma-p2p#186]
**Describe the bug**
When `p2p` tries to commit a compute result that contains an empty state delta, it throws:
```
p2p_1        | [Mon May 06 2019 22:18:29 GMT+0000 (Coordinated Universal Time)] ERROR [COMMIT_RECEIPT] error for task 0x446e739babcd90ddd7f466a78961ca31fa411e14de57f81b359f3cd5d61ecf43 error=  Error: invalid bytes32 value (arg="_stateDeltaHash", coderType="bytes32", value="0x", version=4.0.27)
p2p_1        | (node:38) UnhandledPromiseRejectionWarning: Error: invalid bytes32 value (arg="_stateDeltaHash", coderType="bytes32", value="0x", version=4.0.27)
```
It seems that the value `0x` for an empty value is no longer valid (cc: @apalepu23 )

This is likely a consequence of upgrading web3 to `1.0.0-beta.54` in #178

**To Reproduce**
1. Launch the `discovery-integration-tests`
2. Run the following tests sequentially:
```
01_init.spec.js
02_deploy_flipcoin.spec.js
10_execute_flipcoin.spec.js 
```

**Expected behavior**
The commit receipt should succeed.

**Comment by lenak25** [comment]
P2P doesn't treat empty state delta/output correctly yet. This is addressed in PR #183

**Comment by lacabra** [comment]
I have tested #183 in the network (with only 1 worker), and I confirm that it fixes this issue, especially [this line](module.exports.ETHEREUM_EMPTY_HASH = '0x0000000000000000000000000000000000000000000000000000000000000000';).

This issue can be closed when #183 is merged.

**Comment by lenak25** [comment]
#183 was merged
